src/Browser.php

```diff
@@ -335,4 +337,32 @@ public function withOptions(array $options)
 
         return $browser;
     }
+
+    public function withTimeout($timeout)
```

Comment:
Can you add docblocks with parameter type hints? In particular, should we keep the negative timeout and the `null` timeout value? See also other methods.

Reply:
Don't think so to be honest, only allowing `int` in the type hint